ci/setup-build-test-gdk.ps1

```diff
@@ -54,42 +54,32 @@ class TestSuite {
 [string] $user_cmd_line_args = "$env:TEST_ARGS"
 [string] $gdk_branch = "$env:BUILDKITE_BRANCH"
 
-[TestProjectTarget] $gdk_test_project = [TestProjectTarget]::new("git@github.com:spatialos/UnrealGDKTestGyms.git", $gdk_branch, "Game\GDKTestGyms.uproject", "GDKTestGyms")
-[TestProjectTarget] $native_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NativeNetworkTestProject")
+[TestProjectTarget] $gdk_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NetworkTestProject")
```

Comment:
The alternative is to use the starter template, but it is considerably more work I think.
I would have to setup some custom code to copy the template from the engine, and I am not sure if I can programmatically create a project based on a template.
Though it would offer one advantage in automatically testing our starter template if we could do it.

Reply:
Argh, I just realized another problem - this will destroy the TestGym CI, since the GDK will no longer exercise it.

Reply:
tbf, the only CI we would be getting is that the TestGyms compile...